clang/test/CodeGen/aarch64-sme2-intrinsics/acle_sme2_mls.c

```diff
@@ -246,7 +246,7 @@ void test_svmls_single2_f64(uint32_t slice_base, svfloat64x2_t zn, svfloat64_t z
 // CPP-CHECK-NEXT:    ret void
 //
 void test_svmls_single4_f64(uint32_t slice_base, svfloat64x4_t zn, svfloat64_t zm) __arm_streaming __arm_shared_za {
-  SVE_ACLE_FUNC(svmls_single_za64,,_f64,,_vg1x4)(slice_base, zn, zm);
+  SVE_ACLE_FUNC(svmls,_single,_za64,_f64,_vg1x4)(slice_base, zn, zm);
```

Comment:
I think there are some `svmls` & `svmls_lane` tests that should be updated in this test file as well, for example on line 39:
`SVE_ACLE_FUNC(svmls_za32,,_f32,,_vg1x2)(slice_base, zn, zm);`

Reply:
Is `SVE_ACLE_FUNC(svmls_za32,,_f32,,_vg1x2)(slice_base, zn, zm);` Incorrect? Looking [here](https://github.com/ARM-software/acle/blob/b88cbf7e9c104100bb5016c848763171494dee44/main/acle.md?plain=1#L10228-L10241) it does not appear to be overloaded in the same manner

Reply:
Ah I see what you mean now. Some tests inside `acle_sme2_mlsl.c` were suffering from the same fate too. I've changed them now. Thanks for pointing this out.